scripts/pyinstaller/hooks/hook-pydrive2.py

```diff
@@ -3,4 +3,3 @@
 )
 
 datas = copy_metadata("pydrive2")
```

Comment:
This is still needed since we explicitly need the package metadata for `dvc doctor` output